files/galaxy/dynamic_job_rules/dev/total_perspective_vortex/vortex_config.yml

```diff
@@ -148,3 +155,12 @@ destinations:
         - pulsar-nci-test
       require:
         - pulsar
+  pulsar-eu-gpu-test:
+    cores: 40
+    mem: 110000
+    scheduling:
+      accept:
+        - general
+        - pulsar-eu-gpu-test
+      require:
+        - pulsar
```

Comment:
`pulsar-eu-gpu-test` needs to be in `require` or normal pulsar jobs will be scheduled to this